controllers/mesh_env_config.go

```diff
@@ -33,17 +33,6 @@ func getAuthorinoLabel() ([]string, error) {
 	return keyValue, nil
 }
 
-func getAuthAudience() []string {
-	aud := getEnvOr(AuthAudience, "https://kubernetes.default.svc")
-	audiences := strings.Split(aud, ",")
-
-	for i := range audiences {
-		audiences[i] = strings.TrimSpace(audiences[i])
-	}
-
-	return audiences
-}
-
```

Comment:
So this was introduced specifically to address the case when OpenDataHub runs e.g. on ROSA environment and we need to use third-party token issuers in Istio (so also needs this config change in the cluster:  https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/service_mesh/index#ossm-install-rosa_ossm-create-smcp

We won't need it with the new approach?